gateway/server.go

```diff
@@ -1455,6 +1457,6 @@ func startServer() {
 	mainLog.Info("--> Listening on port: ", config.Global().ListenPort)
 	mainLog.Info("--> PID: ", hostDetails.PID)
 	if !rpc.IsEmergencyMode() {
-		DoReload()
+		reloadURLStructure(func() {})
```

Comment:
we are queuing instead of actual reload here just in case redis is catching up. Eventually the reload happens in the next reload loop tick.